ana_scans.py

```diff
@@ -123,9 +124,9 @@ def launchAnaArgs(anaType, cName, cType, scandate, scandatetrim=None, ztrim=4.0,
   #Execute Commands
   try:
     log = file("%s/anaLog.log"%(dirPath),"w")
-   
-    runCommand(cmd,log)
-    #runCommand(cmd)
+ 
+    #runCommand(cmd,log)
+    runCommand(cmd)
     for key in postCmds:
       runCommand(postCmds[key])
```

Comment:
Implementation requires it to be a dict.

It's a dictionary of lists, runCommand requires a list of strings as inputs to be the command.  We need **multiple** commands to be executed because multiple output files need to be copied, so it's configured as a dictionary.

e.g. syntax of cp command is cp <file> <target> and to keep with the naming scheme each file requires it's own executing of runCommand, so it's own entry in the postCmds dictionary.